install_deps.sh

```diff
@@ -12,10 +12,18 @@ else
     sudo apt-get update
     sudo apt-get install libc6-dev-i386 gcc-multilib g++-multilib
     sudo apt-get install libx11-dev:i386 libxkbfile-dev:i386
+    export CC="/usr/bin/gcc -m32"
```

Comment:
Are these 32-bit changes necessary in this PR? I'm wary to change these if there are no open issues about the 32-bit binaries being broken.

Reply:
I don't really want to be worried by the fact that there may be x86_64 code in the 32-bit versions. This way, we can be sure.

Reply:
That's understandable. For safety, please move these lines to a different PR if you think they are necessary and test the resulting binaries on a 32-bit machine to confirm nothing is broken.